renet/src/transport/native_socket.rs

```diff
@@ -0,0 +1,42 @@
+use std::net::{SocketAddr, UdpSocket};
+
+use super::{NetcodeError, NetcodeTransportError, TransportSocket};
+
+/// Implementation of [`TransportSocket`] for `UdpSockets`.
+#[derive(Debug)]
+pub struct NativeSocket {
+    socket: UdpSocket,
```

Comment:
Maybe implement it directly on `UdpSocket` since the trait in the same crate?
This way it will be less boilerplate to initialize.

Reply:
Ah, I see.